e2e_example/tool/watch.dart

```diff
@@ -17,5 +18,8 @@ Future main() async {
   /// the transitive deps issue.
   CopyBuilder.addPhases(phases, graph);
 
+  /// Adds all the phases necessary to copy all files into a fake packages dir!
```

Comment:
might want to fix existing cases in this file - these aren't doc comments and shouldn't use triple slash